packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -308,6 +318,26 @@ const renderApp = (args) => {
     if (error && isRemote()) {
         delete error.stack
     }
+    performance.mark(PERFORMANCE_MARKS.totalEnd)
+
+    let performanceMetrics = []
+    try {
+        performanceMetrics = getPerformanceMetrics()
+
+        // DISCUSSION! HOW DO WE WANT TO CONTROL THIS?
+        const TBD_FEATURE_FLAG = true
+        if (TBD_FEATURE_FLAG) {
+            console.debug('======Performance metrics======')
+            performanceMetrics.forEach((metric) => {
+                console.debug(`${metric.name} - ${metric.duration}`)
+            })
+            console.debug('===============================')
+        }
```

Comment:
What's the intended use of the performance markers? If it's like a one-time manual-trigger (via the browser), I'd prefer to hide the feature behind a query parameter. 

This query parameter is similar to how we have `?__server_only`.

Reply:
I +1 this comment. If it's not required for rendering to occur then we should not have it run all the time. Using enabling this performance output via query string param or environment variable would be ideal .